packages/x-components/src/x-modules/related-tags/components/__tests__/related-tags.spec.ts

```diff
@@ -114,14 +121,25 @@ describe('testing related tags component', () => {
     expect(labelWrapper.text()).toEqual(relatedTags[relatedTags.length - 1].tag);
     expect(crossWrapper.exists()).toEqual(true);
     unSelectedWrappers.forEach(expectToHaveOverriddenContent);
+
+    for (let i = 0; i < relatedTags.length; i++) {
```

Comment:
**This is just a question**: Why do you change every single related tag to curated? Isn't enough with 1? Tests should be as closer to the real use case as possible.

Reply:
It was an option yes, and I though about it.

I'll replace it.